### PR TITLE
Update instructions for using LFX PCC

### PR DESCRIPTION
**Pull request by jmertic**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207328679729038

### CODE REVIEW

File: tools/collaboration.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Each hosted project and working group has a Public Calendar that can be directly pointed to or embedded in an existing web page. Instructions for getting the link for the hosted project or working group Public Calendar can be found [here][access to public calendar link instructions]. All public meetings for hosted projects and working groups are posted on the [Community Calendar]. 
+
+Meetings that an individual is directly invited to can be found in the 'Meetings' section of that individual's LFX Individual Dashboard; instructions can be found [here][individual dashboard meetings instructions]
+
+## Best Practices
+
+To help individuals not be inundated with calendar overload, try to follow these guidelines:
+
+- Make sure to make meeting invite changes ( including cancellations or reschedulings ) more than 24 hours in advance, as many calendaring clients wait to update. Emailing that calendar's mailing list is also best practice to inform the group of the cancellation.
```
Comment (JeanChristopheMorinPerso) [inline tools/collaboration.md:75]: >Emailing that calendar's mailing list
Should this be cahnged to "Emailing your project's mailing list" instead?

**Review by JeanChristopheMorinPerso — COMMENTED** [review]
Left a small comment, but apart from that, it looks good to me!

Do you think we should document the `calendar.<project-domain>` DNS entries?

File: tools/collaboration.md (hunk 1 above)
Comment (jmertic) [reply tools/collaboration.md]: Cleaned this up - let me know if this reads better

**Comment by jmertic** [comment]
> Do you think we should document the `calendar.<project-domain>` DNS entries?

Yes - added that in there now. Thank you!

**Review by JeanChristopheMorinPerso — COMMENTED** [review]
Looks great!

File: tools/collaboration.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Best Practices
+
+To help individuals not be inundated with calendar overload, try to follow these guidelines:
+
+- Make sure to make meeting invite changes ( including cancellations or reschedulings ) more than 24 hours in advance, as many calendaring clients wait to update. The best practice is to email the respective committee's mailing list or [email the list of meeting attendees in PCC][email meeting participants PCC] to inform the group of the cancellation.
+- Instruct invitees to manually remove old entries when provisioning a new series of recurring meetings. Occasionally, we will see some calendaring clients not removing these automatically.
+- Ensure the meeting description includes an agenda or a link to the agenda.
+
+[Community Calendar]: https://calendar.openmainframeproject.org
```
Comment (matthewlow-dwa) [inline tools/collaboration.md:83]: This link points to the Open Mainframe Project calendar, and should point here instead:
https://calendar.aswf.io

Comment (jmertic) [reply tools/collaboration.md]: Good catch - fixed now

**Pull request merged**